pkgs/desktops/gnome-3/extensions/pop-os-shell/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, nodePackages, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "pop-os-shell";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "shell";
+    rev =  version;
+    sha256 = "sha256-gv9knhG1/HOUsIyPWfBRBpJWg/bfRXbbg/x2QDgSmls=";
+  };
+
+  nativeBuildInputs = [ glib nodePackages.typescript ];
+
+  makeFlags = [ "INSTALLBASE=$(out)/share/gnome-shell/extensions" ];
+
+  meta = with stdenv.lib; {
+    description = "Keyboard-driven layer for GNOME Shell";
+    license = licenses.gpl3;
```

Comment:
`gpl3` is unclear, use `gpl3Plus` or `gpl3Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

Reply:
I'm not entirely sure, which version to use. The license can be found here: https://github.com/pop-os/shell/blob/master/LICENSE. Also on the web i could not really identify what the difference is between gpl3-only and gpl3+. Can you tell me what we have here? @jtojnar

Reply:
Usually, the precise license is listed in README, but that is not the case [here](https://github.com/pop-os/shell#license). Try asking on the upstream issue tracker.

Reply:
The license is [here](https://github.com/pop-os/shell/blob/master/LICENSE)

Reply:
That is just license terms, it needs to be explicitly specified whether the copyright owner allows using latter versions of the license, see the GNU links in the [Discourse threads](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206).

Reply:
IIRC if not listed explicitly it should be GPL3 only, shouldn't it?

Reply:
Yes, that is all that can be inferred without any more info. Though people often intend `-or-later` so it does not hurt to ask.